src/screens/CreatePost/index.js

```diff
@@ -77,6 +77,8 @@ import {getUrl, isContainUrl} from '../../utils/Utils';
 import {getUserId} from '../../utils/users';
 import {requestCameraPermission, requestExternalStoragePermission} from '../../utils/permission';
 import {uploadPhoto} from '../../service/file';
+import ImageUtils from '../../utils/image';
+import ImageCompressionUtils from '../../utils/image/compress';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New utility modules `ImageUtils` and `ImageCompressionUtils` have been imported. Make sure these modules exist and are properly exported from their respective files.
```diff
+ import ImageUtils from '../../utils/image';
+ import ImageCompressionUtils from '../../utils/image/compress';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->